src/js/claims-status/containers/AppealsV2StatusPage.jsx

```diff
@@ -36,29 +38,55 @@ class AppealsV2StatusPage extends React.Component {
   }
 }
 
+AppealsV2StatusPage.defaultProps = {
+  appeal: {
+    id: '',
+    type: '',
+    attributes: {
+      events: [],
+      status: {
+        type: '',
+        details: {}
+      }
+    }
+  }
+};
+
 AppealsV2StatusPage.propTypes = {
   params: PropTypes.shape({
     id: PropTypes.string.isRequired
-  }).isRequired
+  }).isRequired,
+  appeal: PropTypes.shape({
+    id: PropTypes.string.isRequired,
+    type: PropTypes.string.isRequired,
+    attributes: PropTypes.object.isRequired, // Can flesh this out later
+  })
 };
 
-function mapStateToProps(state, ownProps) {
+function isolateAppeal(state, id) {
```

Comment:
Might we want to extract this into helpers or somewhere like that? That would let us reuse this for other components.